### ISSUE TITLE
Sync command results in a panic

### ISSUE THREAD

**Issue by mrsimonemms** [ellie/atuin#1051]
Howdy, I've got a really weird error when I run `atuin sync` which I don't know how to debug - I've run with `RUST_BACKTRACE=full` set to give the full error message

```shell
thread 'main' panicked at 'failed to decrypt history! check your key: array had incorrect length, expected 8

Location:
    atuin-client/src/encryption.rs:101:19', /home/runner/work/atuin/atuin/atuin-client/src/api_client.rs:167:45
stack backtrace:
   0:     0x55d9753a82ca - <unknown>
   1:     0x55d9753cf29e - <unknown>
   2:     0x55d9753a1d75 - <unknown>
   3:     0x55d9753a8095 - <unknown>
   4:     0x55d9753a987f - <unknown>
   5:     0x55d9753a95bb - <unknown>
   6:     0x55d9753a9f8c - <unknown>
   7:     0x55d9753a9d29 - <unknown>
   8:     0x55d9753a877c - <unknown>
   9:     0x55d9753a9a32 - <unknown>
  10:     0x55d9749fe4b3 - <unknown>
  11:     0x55d9749fe943 - <unknown>
  12:     0x55d974abd333 - <unknown>
  13:     0x55d974b64101 - <unknown>
  14:     0x55d974ac615f - <unknown>
  15:     0x55d974adfb4a - <unknown>
  16:     0x55d974aea96f - <unknown>
  17:     0x55d974c290d4 - <unknown>
  18:     0x55d974ba7f12 - <unknown>
  19:     0x55d974c286ee - <unknown>
  20:     0x55d974b19421 - <unknown>
  21:     0x55d974aa9a58 - <unknown>
  22:     0x55d974b8b735 - <unknown>
  23:     0x55d974a5a1a3 - <unknown>
  24:     0x55d974afbc3d - <unknown>
  25:     0x55d97539b0ac - <unknown>
  26:     0x55d974b8e1a5 - <unknown>
  27:     0x7fe4d1a23510 - __libc_start_call_main
                               at ./csu/../sysdeps/nptl/libc_start_call_main.h:58:16
  28:     0x7fe4d1a235c9 - __libc_start_main_impl
                               at ./csu/../csu/libc-start.c:381:3
  29:     0x55d9749feb2e - <unknown>
  30:                0x0 - <unknown>
```

Does anyone have any ideas on how I can best I can debug (or fix) please?

---

Version: atuin 13.0.1
OS: Ubuntu 22.10

**Comment by mrsimonemms** [comment]
Right, I've solved the issue with an upgrade to atuin v15.0.0. I guess it must be a versioning issue (I can't remember what I'm running on my other computer).

Going to put this one down to experience, but hopefully an upgrade will solve the same problem for anyone else experiencing it.

**Comment by mrsimonemms** [comment]
Yeah, my other machine was on v15.0.0 so I guess there was an issue between v13 and v15.

Still, guess that proves you're using [Semantic Versioning](https://semver.org/) correctly 🤣
